doc/goals-non-goals.md

```diff
@@ -46,4 +46,4 @@ that slow self-made IPC to communicate with his sensors because his parents want
 makes it hard for him to react in real-time to dangerous incidents like flying coffee cups. When strolling through
 the interwebs on a lonely evening, he finds out about iceoryx: Free-to-use, high-performance data transfer with low
```

Comment:
Quite the compelling story. Interwebs ... I would keep it professional and not kiddie style.